ui/app/common/util/dateUtil.js

```diff
@@ -270,7 +270,7 @@ Bahmni.Common.Util.DateUtil = {
     },
 
     parseLongDateToServerFormat: function (longDate) {
```

Comment:
This seems to be a method in dateUtils.js (which is a common utility), and hence there will possibly be multiple users of this method. Have you verified that all callers of this method will continue to work fine with this change. Alternatively introduce a new method called, parseLongDateToServerFormatWithTimezone( ) so, only callers which care about this information call it.

Reply:
@gsluthra we did manual check with @gokultw . it seems that this changes doesnot effect, anywhere this method has been used.

Reply:
we identified one issue in appointments but that is an existing issue.

Reply:
@gokultw do we have a card created for the appointments issue?

Reply:
I need to check with CURE team.